Introduce Chapter-3 Introduction Page

shaavan opened the pull request:
Closes #501
- This PR introduces the Intro page and introduces the subsequent updates to progress to keep track of them.
- This PR also temporarily introduces challenge-1 for Chapter-3, which is introduced to check the correct functioning of the "Continue" button. It will be removed before the final merge of PR.

afterburn commented:
@shaavan retargeted your branch to `dev`, please resolve conflicts when you have time, thanks

shaavan commented:
Updated from 70f79e9a115807b777557d1171dae4f55eca24ae to ff3d94c2f8f92c53f3f3618f47b94bcfd765c829 ([pr516.01](https://github.com/shaavan/saving-satoshi/commits/pr516.01) -> [pr516.02](https://github.com/shaavan/saving-satoshi/commits/pr516.02), [diff](https://github.com/shaavan/saving-satoshi/compare/pr516.01..pr516.02))
Address @afterburn comment

Changes:
1. Rebased over `dev`
2. Resolved merge conflicts

afterburn commented on content/lessons/chapter-3/index.ts:
```diff
@@ -0,0 +1,11 @@
+const Chapter3 = {
+  'intro-1': Intro1,
+  'intro-2': Intro2,
+  'you-vs-bit-1': YVB1,
```
Lets rename `you-vs-bit-1` to `solo-1` and for the co-op and pool once we'll add later we can use `coop` and `pool`

shaavan replied:
Sounds good! Let me rename it!

shaavan replied:
Addressed in 5a1178b3952e7d024452fad6416a2a65c9fdba47
Thanks for the suggestion!

afterburn commented on content/lessons/chapter-3/you-vs-bit-1.tsx:
```diff
@@ -0,0 +1,22 @@
+export const metadata = {
+  title: 'chapter_two.intro_one.title',
+  image: '/assets/images/chapter-2-intro-1.jpg',
+  key: 'CH3YVB1',
```
Keys can be `CH3SOL1`, `CH3COO1`, `CH3POO1` (that last one is a but unfortunate lol)

shaavan replied:
Addressed in https://github.com/saving-satoshi/saving-satoshi/commit/5a1178b3952e7d024452fad6416a2a65c9fdba47

afterburn reviewed (changes requested):
Thanks @shaavan, requested some minor changes.

shaavan commented:
Updated from ff3d94c2f8f92c53f3f3618f47b94bcfd765c829 to 5a1178b3952e7d024452fad6416a2a65c9fdba47 ([pr516.02](https://github.com/shaavan/saving-satoshi/commits/pr516.02) -> [pr516.03](https://github.com/shaavan/saving-satoshi/commits/pr516.03), [diff](https://github.com/shaavan/saving-satoshi/compare/pr516.02..pr516.03))
Addressed @afterburn comment

- Renamed `you-vs-bit-1` -> `solo-1`
- Renamed `YVB1` -> `SOL1`
